db/seeds.rb

```diff
@@ -1,7 +1,66 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+class Seed
+  def self.start
+    seed = Seed.new
+    seed.generate_users
+    seed.generate_folders
+    seed.generate_files
+    seed.generate_comments
+    seed.create_admin
+  end
+  
+  def generate_users
+    1000.times do |i|
+      user = User.create!(
+            first_name: Faker::Name.first_name,
+            last_name: Faker::Name.last_name,
+            email: Faker::Internet.email,
+            password: "password",
+            cellphone: Faker::PhoneNumber.cell_phone
+      )
+      puts "User #{i}: #{user.first_name} created!"
+    end
+  end
+
+  def generate_folders
+    1000.times do |i|
+      user = User.find(Random.new.rand(1..1000))
```

Comment:
Instead of using User.find(rand(1..1000), I recommend we use User.all.sample or do a pluck on the ids and sample from that array. It might be slower, but we'll receive an error if it hits an id that doesn't exist.